test/project_types/extension/tasks/execute_commands/serve_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require "test_helper"
+
+module Extension
+  module Tasks
+    module ExecuteCommands
+      class ServeTest < MiniTest::Test
+        def setup
+          super
+          ShopifyCLI::ProjectType.load_type(:extension)
+        end
+
+        def test_error_is_raised_if_error_occurs_on_initialize
+          assert_raises StandardError do
```

Comment:
Similar comment here, where's the `StandardError` coming from? Is it failing because the test tests the integration between `ExecuteCommands` and the system?

Reply:
same as above